src/nvim/buffer.c

```diff
@@ -3570,20 +3570,6 @@ void fname_expand(buf_T *buf, char_u **ffname, char_u **sfname)
   if (*sfname == NULL)          /* if no short file name given, use ffname */
     *sfname = *ffname;
   *ffname = (char_u *)fix_fname((char *)*ffname);     /* expand to full path */
-
-#ifdef FEAT_SHORTCUT
```

Comment:
Very useful info. @Hettomei If you happen to touch the relevant commit again, please add a `References #460.` statement in the commit message.